roles/bwc/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+bwc_pkg_repo: "enterprise"
+# 'latest' to get latest version or numeric like '2.1.1'
+bwc_version: latest
+# used only if 'bwc_version' is numeric
+bwc_revision: 1
+
+st2_config_file_path: /etc/st2/st2.conf
```

Comment:
As noted before, don't forget to hardcode `st2.conf` path.
We don't need it in vars (https://github.com/StackStorm/ansible-st2/pull/126#discussion_r104986082 reasons why).